indico/modules/events/registration/util.py

```diff
@@ -185,3 +185,17 @@ def generate_csv_from_registrations(registrations, regform_items, special_items)
         writer.writerow(registration_dict)
     buf.seek(0)
     return buf
+
+
+def get_user_complete_registrations(user, event):
```

Comment:
Ok, I thought it could be quite useful, but nevermind I can merge these two functions into one.